lib/rdoc/rdoc.rb

```diff
@@ -507,6 +514,26 @@ def document options
     exit @stats.fully_documented? if @options.coverage_report
   end
 
+  def start_server
+    begin
+      require 'webrick'
+    rescue LoadError
+      abort "webrick is not found. You may need to `gem install webrick` to install webrick."
+    end
+
+    # Change the output directory to tmp so it doesn't overwrite the current documentation
+    Dir.chdir "tmp" do
```

Comment:
Discussed this in Slack. Let's look at utilizing `mktmpdir` instead of `$CWD/test`